miio/airpurifier_miot.py

```diff
@@ -249,7 +277,146 @@ def __repr__(self) -> str:
         return s
 
 
-class AirPurifierMiot(MiotDevice):
+class AirPurifierMB4Status(BasicAirPurifierMiotStatus):
+    """
+    Container for status reports from the  Mi Air Purifier 3C (zhimi.airpurifier.mb4).
+
+    {
+        'power': True,
+        'mode': 1,
+        'aqi': 2,
+        'filter_life_remaining': 97,
+        'filter_hours_used': 100,
+        'buzzer': True,
+        'led_brightness_level': 8,
+        'child_lock': False,
+        'motor_speed': 392,
+        'favorite_rpm': 500
+    }
+
+    Response (MIoT format)
+
+    [
+        {'did': 'power', 'siid': 2, 'piid': 1, 'code': 0, 'value': True},
+        {'did': 'mode', 'siid': 2, 'piid': 4, 'code': 0, 'value': 1},
+        {'did': 'aqi', 'siid': 3, 'piid': 4, 'code': 0, 'value': 3},
+        {'did': 'filter_life_remaining', 'siid': 4, 'piid': 1, 'code': 0, 'value': 97},
+        {'did': 'filter_hours_used', 'siid': 4, 'piid': 3, 'code': 0, 'value': 100},
+        {'did': 'buzzer', 'siid': 6, 'piid': 1, 'code': 0, 'value': True},
+        {'did': 'led_brightness_level', 'siid': 7, 'piid': 2, 'code': 0, 'value': 8},
+        {'did': 'child_lock', 'siid': 8, 'piid': 1, 'code': 0, 'value': False},
+        {'did': 'motor_speed', 'siid': 9, 'piid': 1, 'code': 0, 'value': 388},
+        {'did': 'favorite_rpm', 'siid': 9, 'piid': 3, 'code': 0, 'value': 500}
+    ]
+
+    """
+
+    @property
+    def led_brightness_level(self) -> int:
+        """Return brightness level."""
+        return self.data["led_brightness_level"]
+
+    @property
+    def favorite_rpm(self) -> int:
+        """Return favorite rpm level."""
+        return self.data["favorite_rpm"]
+
+    def __repr__(self) -> str:
+        s = (
+            "<AirPurifierMiotStatus power=%s, "
+            "aqi=%s, "
+            "mode=%s, "
+            "led_brightness_level=%s, "
+            "buzzer=%s, "
+            "child_lock=%s, "
+            "filter_life_remaining=%s, "
+            "filter_hours_used=%s, "
+            "motor_speed=%s, "
+            "favorite_rpm=%s>"
+            % (
+                self.power,
+                self.aqi,
+                self.mode,
+                self.led_brightness_level,
+                self.buzzer,
+                self.child_lock,
+                self.filter_life_remaining,
+                self.filter_hours_used,
+                self.motor_speed,
+                self.favorite_rpm,
+            )
+        )
+        return s
+
+
+class BasicAirPurifierMiot(MiotDevice):
+    """Main class representing the air purifier which uses MIoT protocol."""
+
+    def __init__(
+        self,
+        attributes: dict = [],
+        ip: str = None,
+        token: str = None,
+        start_id: int = 0,
+        debug: int = 0,
+        lazy_discover: bool = True,
+    ) -> None:
+        super().__init__(attributes, ip, token, start_id, debug, lazy_discover)
```

Comment:
```suggestion
```
I don't think this is necessary here, as it does nothing calling the parent class `__init__` would do?